GSoC-2025/pranav_kapoor.md

```diff
@@ -0,0 +1,80 @@
+## Personal details
+- **Full name:** Pranav Kapoor  
+- **Email:** pranav33317@gmail.com  
+- **GitHub username:** pranav33317  
+- **Zulip username:** Pranav Kapoor  
+- **Location & time-zone:** New Delhi, India (IST, GMT+5:30)  
+- **Personal website / project portfolio:** [www.github.com/pranav33317](https://www.github.com/pranav33317)  
+- **Code contribution:**  
+  - PR: [https://github.com/brainglobe/brainglobe-atlasapi/pull/546#pullrequestreview-2729541724](https://github.com/brainglobe/brainglobe-atlasapi/pull/546#pullrequestreview-2729541724)  
+    I tried to disable parallel mesh creation in various files to simplify mesh creation while simultaneously working on [Issue #429](https://github.com/brainglobe/brainglobeatlasapi/issues/429).  
+- **Proposal discussion link:**  
+  - [https://github.com/brainglobe/brainglobe-atlasapi/pull/554](https://github.com/brainglobe/brainglobe-atlasapi/pull/554)  
+    This is a closed PR where I created a new file containing the wrapper function to convert an atlas to an OpenSANDS object and its file path.
+
+---
+
+## Project proposal
+
+### Synopsis
+As someone deeply fascinated by the intricate workings of the brain, I have always admired how neuroanatomical atlases empower researchers to navigate complex brain structures with ease. BrainGlobe’s Atlas API stands out as a robust tool that consolidates atlas‑registered data into a unified Python interface. There is now an exciting opportunity to further enhance this system by adopting the OpenMINDS SANDS standard—a community‑driven format that delivers more interoperable brain data. This upgrade will enable researchers to derive deeper, more accurate insights.
+
+My project aims to enhance the utility of the BrainGlobe ecosystem by updating the atlas packaging scripts so they output data in the OpenMINDS SANDS format. This upgrade is critical for researchers who need consistent, comprehensive data to integrate findings across studies and species. By delivering a modular, well-documented solution complete with rigorous testing, I will enable both seasoned neuroscientists and new entrants in the field, making advanced neuroinformatics more accessible to all.
+
+### Implementation timeline
+
+1. **Minimal Deliverables:**
+   - Modify existing atlas packaging code to output atlas data conforming to the OpenMINDS SANDS standard.
+   - Adapt at least one packaging script (e.g., the Allen Human atlas script) to incorporate this new functionality.
+   - Develop comprehensive tests covering all new and modified features.
+   - Update the documentation to guide users on how to generate and utilize OpenMINDS SANDS‑compliant atlases.
+2. **Stretch Goals (if time permits):**
+   - Extend support for additional species or imaging modalities.
```

Comment:
Do you mean extend support for additional atlases? Imaging modalities isn't really relevant here.